code/go/internal/validator/test/packages/good/data_stream/ilm_policy/manifest.yml

```diff
@@ -0,0 +1,7 @@
+title: Hidden data stream and ilm policy overrride
+type: metrics
+ilm_policy: diagnostics
```

Comment:
Thanks for adding test case. If I understand the idea correctly, the diagnostics should be present in this package somewhere? If so, we can try to enable validation for the property (mentioned in other comment).

Reply:
@mtojek the diagnostics is referring to a use case for a particular `ilm_policy` that we want to create.  

Our use case is to enable ourselves to add a custom `ilm_policy` for a `data stream` in a package.  We then want to define that `ilm_policy` to be fairly aggressive for a `data stream` that is intended to be for documents used for diagnostics/telemetry.  A separate telemetry server will query for the docs stored in this `data stream`.

Reply:
Thanks Kevin for the explanation! It's more clear now. Just a follow-up question: based on this example (diagnostics) do you plan to include it in the package or it will be preinstalled/predefined/distributed with ES or Kibana? If it's distributed together with the package, then we can consider enabling a validation for the field (I'm fine with leaving it for later improvement).

Reply:
@mtojek we'll include it with the package.  As such, I think we should eventually add a validation for it.  We see it as fairly similar to the ingest_pipeline.  I opened a follow up ticket for adding that validation - https://github.com/elastic/package-spec/issues/92 (I can fill it out with more details)

Reply:
SGTM!